.circleci/workflows.yml

```diff
@@ -38,22 +37,23 @@ executors:
 jobs:
   build:
     docker: &docker
-      - image: python:<< pipeline.parameters.python-version >>
+      - image: cimg/python:<< pipeline.parameters.python-version >>
```

Comment:
Using the convience image supposedly also provides [faster spin-up times](https://circleci.com/docs/guides/execution-managed/circleci-images/). Based on my tests, the difference was not very noticeable.